### DISCUSSION TITLE
v3版本如何配置直链

### DISCUSSION THREAD

**Discussion by yqchilde** [AlistGo/alist#1688] [Q&A] [answered]
在v2时没有`sign`参数，我会用直链做一些持久的下载，现在v3版中有了sign，除了延长缓存时间，是否有单独设置不需要sign的方法🙏🏻

**Comment by xhofe** [comment, marked as answer]
不添加密码就不需要sign。
或者 直链有效期 = 0 sign就不会过期

**Reply by yqchilde** [reply]
已解决，确实是由于我元信息设置密码并应用到子文件夹，感谢作者这么快回复！！

**Reply by ianniu** [reply]
> 不添加密码就不需要sign。 或者 直链有效期 = 0 sign就不会过期

您好，请问不添加密码指的是不添加元信息密码么？我没有添加元信息密码但下载链接中还是会有sign
**解决了，在设置->全局中关闭签名所有**
